GCC 3.3.5 has a bug in the i386-elf target: it fails to emit
.intel_syntax in assembly files if -masm=intel is passed on the
command line.  This is because elfos.h overrides the ASM_FILE_START
provided by gcc/config/att.h, which emits that directive, with a
version that does not.  This patch covers up the problem.

--- gcc/config/elfos.h~ 2005-01-03 21:14:58.312309000 -0800
+++ gcc/config/elfos.h  2005-01-03 21:03:51.758598000 -0800
@@ -97,9 +97,11 @@
    directive for any specific target, you should override this definition
    in the target-specific file which includes this one.  */

+#if 0
 #undef ASM_FILE_START
 #define ASM_FILE_START(FILE)                            \
   output_file_directive ((FILE), main_input_filename)
+#endif

 /* This is how to allocate empty space in some section.  The .zero
    pseudo-op is used for this on most svr4 assemblers.  */
